### PR TITLE
Add note regarding default behavior of sqlite dirs

### PR DESCRIPTION
**Pull request by meierale**
As suggested by @jorgeorpinel, I moved the default behavior description of `md5s` and `links`dirs to here. (from https://github.com/iterative/dvc.org/pull/2776)

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
OK thanks! Could've just included the changes in #2776 but this is fine too 🙂

File: content/docs/user-guide/project-structure/internal-files.md [hunk 1]
```diff
@@ -62,6 +62,13 @@ operation.
 
 - `.dvc/tmp/exps`: This directory will contain workspace copies used for
   temporary or parallel <abbr>experiments</abbr> (see `dvc exp run`).
+  
+<a name="sqlite-default">Note 1</a>: To avoid any possible state corruption,by default the SQLite-based 
+remote indexes and state's `links` and `md5s` are cached internally in the 
+repository itself. This may however be overriden if necessary,
```
Comment (jorgeorpinel) [inline content/docs/user-guide/project-structure/internal-files.md:68]: > the SQLite-based remote indexes

So this note should apply to `.dvc/tmp/index` too?

Comment (meierale) [reply content/docs/user-guide/project-structure/internal-files.md:68]: I actually have to double check again but if I'm not mistaken it applies to all three (index, links and md5s) but is controlled by just the 2 config settings.

**Comment by jorgeorpinel** [comment]
Thanks again for this @meierale. Closing for now per https://github.com/iterative/dvc.org/pull/2776#issuecomment-922362267 (cc @efiop) but please reopen if I'm mistaken and I'll wrap this one up as well.

**Comment by jorgeorpinel** [comment]
Reopening per https://github.com/iterative/dvc.org/pull/2776#issuecomment-934078091

**Review by jorgeorpinel — APPROVED** [review]
Clarified, copy edited, and formatted this info. Merging to simplify the process since this is from a fork...

**Comment by jorgeorpinel** [comment]
Thank you BTW, @meierale !

**Comment by jorgeorpinel** [comment]
Merged but I'm not sure it's correct.

@efiop can you take a look please? I'll address in a separate PR.

File: content/docs/command-reference/config.md [hunk 2]
```diff
@@ -223,34 +223,33 @@ connection settings, and configuring a remote is the way that can be done.
...
+  the physical size of the state files, as well as the performance of certain
+  DVC operations. The default is 10,000,000 rows. The bigger the limit, the
+  longer the file hash history that DVC can keep, for example.
+
+- `state.row_cleanup_quota` - percentage of the state database to be deleted
+  when it reaches the `state.row_limit`. The default quota is 50%. DVC removes
+  the oldest entries (created when `dvc status` is used, for example).
+
+- `state.dir` - specify a custom location for the state databases (directories),
+  by default `.dvc/tmp/index` and `.dvc/tmp/md5`. This may be necessary when
```
Comment (efiop) [inline content/docs/command-reference/config.md:240]: `.dvc/tmp/index` -> `.dvc/tmp/links`

**Pull request merged**